client/js/preload/goldstoneRouter.js

```diff
@@ -37,6 +37,7 @@ var LauncherView = Backbone.View.extend({
 var GoldstoneRouter = Backbone.Router.extend({
     routes: {
         "api_perf/report": "apiPerfReport",
+        "apps/opentrail": "openTrail",
```

Comment:
@lexjacobs: Does the /applications/ API need to return more (or different) data in order to do this?